File: 2026-08/src/day_29/prms_120891/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(int order) {
    int cnt = 0;
    while (order > 0) {
        int d = order % 10;
        if (d == 3 || d == 6 || d == 9) cnt++;
        order /= 10;
    }

    return cnt;
}

File: 2026-08/src/day_29/prms_120891/Solution.java
package day_29.prms_120891;

class Solution {
    public int solution(int order) {
        int cnt = 0;
        while (order > 0) {
            int r = order % 10;
            if (r == 3 || r == 6 || r == 9) cnt++;
            order /= 10;
        }

        return cnt;
    }
}
